resources/carts.raml

```diff
@@ -2,7 +2,7 @@ type:
   baseDomain:
     resourceType: Cart
     # change to: resourceQueryType: CartPagedQueryResponse | Cart
-    resourceQueryType: CartPagedQueryResponse | object
+    resourceQueryType: CartPagedQueryResponse
```

Comment:
the object has to used in order to allow the Cart return type which you will get in case the customerId query parameter is set.